src/test/java/ValidateTest.java

```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
         Assert.assertNotNull(getFileContent("users.json"));
     }
 
+
+    static private  gaAnswer getUsersAnswer(){
```

Comment:
\[Checkstyle\] ERROR: 'private' modifier out of order with the JLS suggestions\.